.vitepress/config.ts

```diff
@@ -664,33 +610,99 @@ function sidebarHome() {
                         },
                       ],
                     },
+                    
                   ],
                 },
+                {
+                  text: "Ethereum fallback mechanism",
+                  link: "/developers/ethereum-fallback",
+                },
               ],
             },
-            { text: "Rollkit", link: "/developers/rollkit" },
             {
-              text: "Astria",
+              text: "Sovereign",
+              collapsed: true,
+              items: [
+              {
+                text: "Dymension",
+                link: "https://dymension.xyz/",
+              },
+              { text: "Rollkit", link: "https://rollkit.dev" },
+              {
+                text: "Sovereign SDK",
+                link: "https://github.com/Sovereign-Labs/sovereign-sdk/tree/stable/examples/demo-rollup#demo-rollup",
+              },
+              {
+                text: "Vistara",
+                link: "https://docs.vistara.dev/",
```

Comment:
remove

Reply:
got it. will remove button on MD page too. however, it's listed on our site still on https://celestia.org/build/

Reply:
https://github.com/celestiaorg/docs/pull/1515/commits/c6e53a38ee4fc6cef432c08547899907137e1169

Reply:
fixed in https://github.com/celestiaorg/docs/pull/1515/commits/2436703eb9503de0273bf6fba2999b498abafc18